src/dodal/devices/oav/pin_image_recognition/utils.py

```diff
@@ -14,6 +14,12 @@ class ScanDirections(Enum):
     REVERSE = -1
 
 
+"""
+See https://opencv24-python-tutorials.readthedocs.io/en/latest/py_tutorials/py_imgproc/py_morphological_ops/py_morphological_ops.html 
+for description of functions below.
+"""
```

Comment:
Could: This seems unrelated so technically shouldn't be part of this PR but it's just some docs so fine to keep here